crates/fluvio-cli/src/connector/update.rs

```diff
@@ -0,0 +1,87 @@
+//!
+//! # Update a Managed Connector
+//!
+//! CLI tree to generate Update a Managed Connector
+//!
+
+use fluvio::metadata::topic::ReplicaSpec;
+use structopt::StructOpt;
+use tracing::debug;
+
+use fluvio::{Fluvio, FluvioError};
+use fluvio::metadata::{
+    topic::{TopicSpec, TopicReplicaParam},
+    connector::ManagedConnectorSpec,
+};
+
+use fluvio_sc_schema::ApiError;
+use fluvio_sc_schema::errors::ErrorCode;
+
+use crate::CliError;
+use super::ConnectorConfig;
+
+// -----------------------------------
+// CLI Options
+// -----------------------------------
+
+#[derive(Debug, StructOpt, Default)]
+pub struct UpdateManagedConnectorOpt {
+    /// The name for the update Managed Connector
```

Comment:
I think that we should update this and also for the create option
```suggestion
    /// The filename with the configurations to update Managed Connector
```